docs/changes/2381.feature.rst

```diff
@@ -0,0 +1 @@
+Added new fields to the `MuonEfficiencyContainer` - `is_valid` to check if fit converged successfully, `parameters_at_limit` to check if parameters were fitted close to a bound and `likelihood_value` which represents cost function value atthe minimum. These fields were added to the output of the `MuonIntensityFitter`
```

Comment:
```suggestion
Added new fields to the ``MuonEfficiencyContainer`` - ``is_valid`` to check if fit converged successfully, ``parameters_at_limit`` to check if parameters were fitted close to a bound and ``likelihood_value`` which represents cost function value at the minimum. These fields were added to the output of the ``MuonIntensityFitter``
```

In restructured text, you need to use two back-ticks for code, a single back-tick is interpreted as a reference and the docs builds now complains about undefined references.

So you need to repace all single ticks here with double ticks.